expr/functions/divideSeries/function_test.go

```diff
@@ -131,11 +131,7 @@ func TestDivideSeriesAligned(t *testing.T) {
 	for _, tt := range tests {
 		testName := tt.Target
 		t.Run(testName, func(t *testing.T) {
-			err := th.TestEvalExprModifiedOrigin(t, &tt, 0, 1, false)
-			if err != nil {
-				t.Errorf("unexpected error while evaluating %s: got `%+v`", tt.Target, err)
-				return
-			}
+			th.TestEvalExpr(t, &tt)
```

Comment:
Why remove ?  It's for checks missed copy-on-write.

Reply:
Resolving this, please feel free to re-open if needed.